return assert(require "lp.utils.t.error_managment".on_error)
